<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CClient</class>
 <widget class="QDialog" name="CClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>239</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CClient</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="hostLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&amp;Server name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="hostCombo"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QLineEdit" name="portLineEdit"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="portLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>S&amp;erver port:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="statusLabel">
     <property name="text">
      <string>本模块需要服务端模块处于运行状态。</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>232</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1" rowspan="4">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="btnConnect">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSetData">
       <property name="text">
        <string>Set Data</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnGetData">
       <property name="text">
        <string>Get Data</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnQuit">
       <property name="text">
        <string>Quit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>id:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leSetDataId"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>value:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leSetDataValue"/>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>id:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leGetDataId"/>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>value:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leGetDataValue">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>232</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
